sbin/jtreg_prep.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Purpose: This script was designed to do any+all setup required by the jtreg.sh script in order to run it.
+# Tasks: Retrieve Java, unpack it if need-be, and store it locally in a specific location. If the location is blank, we put it in 
+
+REPOSITORY=AdoptOpenJDK/openjdk-jdk8u
+OPENJDK_REPO_NAME=openjdk
+
+while [[ $# -gt 0 ]] && [[ ."$1" = .-* ]] ; do
```

Comment:
I think you can do `[[ ."$1" = .-* ]]` -> `[[ $1 == -* ]]`. Also probably easier to use `==` here (for consistency with the rest of this script).